PopPUNK/assign.py

```diff
@@ -408,6 +408,11 @@ def assign_query_hdf5(dbFuncs,
         raise RuntimeError("lineage models cannot be used with --serial")
     model.set_threads(threads)
 
+    # Only proceed with a fully-fitted model
+    if not model.fitted or (hasattr(model,'assign_points') and model.assign_points == False):
+        sys.stderr.write('Cannot assign points with an incompletely-fitted model\n')
```

Comment:
Better message is perhaps possible, i.e. giving the suggested fix 're-run without --no-assign' or similar

Reply:
Good point, edited in 3f1ea31, alternative suggestions welcome.